src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -69,7 +69,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_NativeExecutableExtension Condition="'$(_NativeExecutableExtension)' == '' and $(RuntimeIdentifier.StartsWith('win'))">.exe</_NativeExecutableExtension>
 
     <_DotNetHostExecutableName>dotnet$(_NativeExecutableExtension)</_DotNetHostExecutableName>
-    <_DotNetAppHostExecutableName>apphost$(_NativeExecutableExtension)</_DotNetAppHostExecutableName>
+    <_DotNetAppHostExecutableNameWithoutExtension>apphost</_DotNetAppHostExecutableNameWithoutExtension>
```

Comment:
I'm trying to understand this change.  It seems the only place we use `_DotNetAppHostExecutableNameWithoutExtension` is where there's a check to append `.exe` if on Windows.  Isn't that `_DotNetAppHostExecutableName`?

Reply:
This change it is to get a clean string "apphost" and it is only in one place.  The problem I cannot use "_DotNetAppHostExecutableName" is even on Windows, GenerateShims can create Linux/mac apphost. But at that point, it is apphost.exe passed in.

Reply:
Oh, I see.  It's using a different RID list to check.  Makes sense.